views/fe-utils/fetchUtil.js

```diff
@@ -19,49 +19,52 @@ const fetchStudies = async () => {
     method: 'GET',
   })
   if (res.status !== 200) {
-    throw new Error(res.statusText);
+    throw new Error(res.statusText)
   }
-  return res.json();
+  return res.json()
 }
 
 const fetchSubjects = async () => {
   const userAccess = await window.fetch(`${basePath}/api/v1/studies`, {
     ...defaultApiOptions,
     method: 'GET',
-  });
+  })
   if (userAccess.status !== 200) {
-    throw new Error(userAccess.statusText);
+    throw new Error(userAccess.statusText)
   }
-  const studiesJson = await userAccess.json();
-  const studies = studiesJson ? studiesJson : [];
-  const subjectsResponse = await window.fetch(`${basePath}/api/v1/subjects?q=${JSON.stringify(studies)}`, {
-    ...defaultApiOptions,
-    method: 'GET',
-  })
+  const studiesJson = await userAccess.json()
+  const studies = studiesJson ? studiesJson : []
+  const subjectsResponse = await window.fetch(
+    `${basePath}/api/v1/subjects?q=${JSON.stringify(studies)}`,
```

Comment:
should we update this to use route constants?

```suggestion
    `${routes.subjects}?q=${JSON.stringify(studies)}`,
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/118/commits/a02b5f9495eed90745bb37c6f90df4df8e963870

There are other routes that could be updated with this refactor. I could do update them on this pr or another pr. Thoughts?